python-sdk/example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -75,10 +76,22 @@ def example_amazon_s3_snowflake_transform():
     temp_table_1 = aql.load_file(
         input_file=File(path=f"{s3_bucket}/ADOPTION_CENTER_1_unquoted.csv"),
         output_table=input_table_1,
+        load_options=[
```

Comment:
could you add documentation for these changes?

```

- Add validation mode as part of the COPY INTO command. Specify the supported validation mode; RETURN_n_ROWS or RETURN_ERRORS or RETURN_ALL_ERRORS. This instructs the COPY command to validate the data files instead of loading them into the specified table; i.e. the COPY command tests the files for errors but does not load them. Read more at:
https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html#optional-parameters
- Remove the ON_ERROR=Continue hardcoded for CSV file types
- Add an option for the user to pass ON_ERROR=Continue as part of copy_options

```

Reply:
<img width="881" alt="Screenshot 2023-02-02 at 8 52 41 PM" src="https://user-images.githubusercontent.com/8670962/216361911-5efe5ac3-fa23-4099-b5a5-e74d06ad25ba.png">
This is already added as part of the doc-string and is visible for `SnowflakeLoadOptions`

Reply:
https://github.com/astronomer/astro-sdk/pull/1689/commits/832a3a61e5166a5fe31bfe26f102f06f3951062c